crates/typst/src/layout/place.rs

```diff
@@ -136,3 +146,35 @@ impl Behave for Packed<PlaceElem> {
         Behaviour::Ignorant
     }
 }
+
+/// Asks the layout algorithm to place pending figures before
+/// continuing with the content.
+///
+/// This is useful for preventing floating figures from spilling
+/// into the next section.
+///
+/// ```example
+/// #set page(height: 150pt, width: 150pt)
+///
+/// #let floater = place(auto, float: true, rect(width: 100%, height: 90pt, text(size: 24pt)[I float!]))
+///
+/// Some introductory text.
+///
+/// #floater #floater #floater #floater
+///
+/// Some additional text.
+///
+/// #place.flush()
+///
+/// Some conclusive text.
```

Comment:
The example is a bit long (and also wide, as in the number of columns) for the documentation. Could you try to find something shorter and nicer looking that demonstrates the purpose? Maybe also without a `floater` variable since that complicates things a bit.

Reply:
I’ve changed the example to the `place-figure-flush` test (before I switched the cylinder out with something else).